.github/actions/build-single-project/action.yml

```diff
@@ -38,6 +38,9 @@ runs:
     - name: "Install Cmake"
       shell: bash
       run: echo "yes" | $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --install "cmake;3.22.1"
+    - name: "Install Android SDK Build-Tools"
+      shell: bash
+      run: echo "yes" | $ANDROID_SDK_ROOT/cmdline-tools/latest/bin/sdkmanager --install "build-tools;34.0.0-rc3"
```

Comment:
why do we need this if there is already a constant for it?

Reply:
What do you mean by a constant? This is to install in the workflow runner.